docs/docs-ref-autogen/excel/excel.customxmlpartcollection.yml

```diff
@@ -160,23 +159,22 @@ items:
           ```typescript
           await Excel.run(async (context) => {
               const settings = context.workbook.settings;
-              const xmlPartIDSetting = settings.getItem("ContosoReviewXmlPartId").load("value");
-
+              const xmlPartIDSetting = settings.getItemOrNullObject("ContosoReviewXmlPartId").load("value");
               await context.sync();
-              
+
+          if (xmlPartIDSetting.value) {   

```

Comment:
This entire `if` block needs to be indented (within the `await Excel.run...` block).